src/condition.rs

```diff
@@ -41,3 +41,37 @@ impl From<u8> for Condition {
         }
     }
 }
+
+// Control bits condition
```

Comment:
Is this documentation? Because instead just remove it, doesn't add any relevant information

Reply:
Yes, it is. I will proceed to correct it with `///`.